File: test/samples/sample.lua
function classify(a, b)
  if a > 0 and b > 0 then
    return 1
  elseif a < 0 then
    return 2
  else
    return 3
  end
end

function loop(n, m)
  local total = 0
  for i = 1, n do
    while total < m do
      if total == 7 then
        total = total + 2
      end
      total = total + 1
    end
  end
  return total
end

File: test/samples/sample.rs
fn classify(a: i32, b: i32) -> i32 {
    if a > 0 && b > 0 {
        1
    } else if a < 0 {
        2
    } else {
        3
    }
}

fn looper(n: i32, m: i32) -> i32 {
    let mut total = 0;
    for _i in 0..n {
        while total < m {
            if total == 7 {
                total = total + 2;
            }
            total = total + 1;
        }
    }
    total
}
